conduit/data/datasets/tabular/dummy.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+import numpy as np
+import pandas
+import pandas as pd
+import torch
+
+from conduit.data.datasets.tabular import CdtTabularDataset
+
+
+class RandomTabularDataset(CdtTabularDataset):
+    def __init__(
+        self,
+        num_disc_features: int = 3,
+        num_cont_features: int = 5,
+        num_samples: int = 256,
+        seed: int = 0,
+    ):
+        rng = np.random.default_rng(seed)
+        feats_dict = {}
+        feature_groups: list[slice] = []
+        for i in range(num_disc_features):
+            num_classes = int(rng.integers(low=2, high=10, size=1)[0])
```

Comment:
```suggestion
            num_classes = int(rng.integers(low=2, high=10, size=1)[0])
```
Maybe allow the number of discrete/continuous features to be directly parameterized (with random-sampling being the default)?

Reply:
oh never mind